fluster/fluster.py

```diff
@@ -532,6 +532,73 @@ def _profile_stats(
 
             return output
 
+        def _generate_global_summary(results: Dict[str, List[Tuple[Decoder, TestSuite]]]) -> str:
+            if not results:
+                return ""
+
+            all_decoders = []
+            decoder_names = set()
+            for test_suite_results in results.values():
+                for decoder, _ in test_suite_results:
+                    if decoder.name not in decoder_names:
+                        all_decoders.append(decoder)
+                        decoder_names.add(decoder.name)
+
+            decoder_totals = {dec.name: {"success": 0, "total": 0} for dec in all_decoders}
+            decoder_times = {dec.name: 0.0 for dec in all_decoders}
+            global_profile_stats: Dict[str, Dict[str, Dict[str, int]]] = {dec.name: {} for dec in all_decoders}
+
+            for test_suite_results in results.values():
+                for decoder, test_suite in test_suite_results:
+                    totals = decoder_totals[decoder.name]
+                    totals["success"] += test_suite.test_vectors_success
+                    totals["total"] += len(test_suite.test_vectors)
+
+                    timeouts = (
+                        sum(
+                            ctx.timeout
+                            for tv in test_suite.test_vectors.values()
+                            if tv.test_result == TestVectorResult.TIMEOUT
+                        )
+                        if ctx.jobs == 1
+                        else 0
+                    )
+                    decoder_times[decoder.name] += test_suite.time_taken - timeouts
+
+                    for test_vector in test_suite.test_vectors.values():
+                        if test_vector.profile is not None:
+                            profile_name = test_vector.profile.name
+                            stats = global_profile_stats[decoder.name].setdefault(
+                                profile_name, {"success": 0, "total": 0}
+                            )
+                            stats["total"] += 1
+                            if test_vector.test_result == TestVectorResult.SUCCESS:
+                                stats["success"] += 1
+
+            separator = f"\n|-|{'-|' * len(all_decoders)}"
+            output = "\n# GLOBAL PROFILE SUMMARY"
+            output += "\n|Test|" + "".join(f"{dec.name}|" for dec in all_decoders) + separator
```

Comment:
Since we don't have a list of tests here i would do the following
```suggestion
            output += "\n|TOTALS|" + "".join(f"{dec.name}|" for dec in all_decoders) + separator
```

Reply:
done